### DISCUSSION TITLE
What's the best use case for Bit?

### DISCUSSION THREAD

**Discussion by zol** [teambit/bit#7061] [General]
Hi folks,

I'm trying to wrap my head around around using Bit - it's quite vast and complex from the docs. Is there a specific use case that Bit would be the best for? What are people here using it for? And, is there a showcase somewhere of things that are built with it?

**Comment by ranm8** [comment]
Few examples to why people are using to Bit-
- Build apps. (https://bit.dev/docs/getting-started/composing/create-apps)
- Reuse components. (https://bit.dev/docs/getting-started/collaborate/install-components)
- Build composable platforms. (https://bit.dev/docs/composable-platforms)
- Build design systems. (https://bit.dev/docs/quick-start/design-system)
- Manage a monorepo / multi-repo environment at ease. (https://bit.dev/docs/thinking-bit)

Please let us know if you have further questions!
